packages/client/src/forms/configuration/default/birth.ts

```diff
@@ -19,6 +19,31 @@ import {
   birthDocumentTypeFhirMapping
 } from '@client/forms/register/fieldMappings/birth/mutation/documents-mappings'
 
+const nidIntegrationConditionals = {
+  hideIfNidIntegrationEnabled: {
+    action: 'hide',
+    expression: `const nationalIdSystem =
+        offlineCountryConfig &&
+        offlineCountryConfig.systems.find(s => s.type === 'NATIONAL_ID');
```

Comment:
@naftis  These conditional needs to change to be exclusive to the MOSIP National ID type that is created in this PR: https://github.com/opencrvs/opencrvs-core/pull/4880